src/handlers/frontend.py

```diff
@@ -283,22 +120,13 @@ def get(self, **kwargs):
     (r"/login/?", LoginHandler),
     (GITHUB_CALLBACK_PATH, GithubLoginHandler),
     (r"/logout/?", LogoutHandler),
-    (r"/registry/(.+)/?", RegistryHandler),
-    (r"/registry/?", RegistryHandler),
-    (r"/ui/(.+)/?", SwaggerUIHandler),
-    (r"/ui/?", SwaggerUIHandler),
-    (r"/editor/(.+)/?", APIEditorHandler),
-    (r"/editor/?", APIEditorHandler),
-    (r"/portal/translator/metakg/?", MetaKGHandler),
-    (r"/portal/([^/]+)/?", PortalHandler),
-    (r"/add_api/?", TemplateHandler, {"filename": "reg_form.html"}),
-    (r"/documentation/?", TemplateHandler, {"filename": "documentation.html"}),
-    (r"/dashboard/?", TemplateHandler, {"filename": "dashboard.html"}),
-    (r"/about/?", TemplateHandler, {"filename": "about.html"}),
-    (r"/faq/?", TemplateHandler, {"filename": "faq.html"}),
-    (r"/privacy/?", TemplateHandler, {"filename": "privacy.html"}),
-    (r"/branding/?", TemplateHandler, {"filename": "brand.html"}),
-    (r"/guide/?", TemplateHandler, {"filename": "guide.html"}),
-    (r"/portal/translator/summary?", TemplateHandler, {"filename": "portal-summary.html"}),
-
+    (r"/css/(.*)",tornado.web.StaticFileHandler, {"path": STATICPATH + '/css'}),
+    (r"/js/(.*)",tornado.web.StaticFileHandler, {"path": STATICPATH + '/js'}),
+    (r"/img/(.*)",tornado.web.StaticFileHandler, {"path": STATICPATH + '/img'}),
+    (r"/fonts/(.*)",tornado.web.StaticFileHandler, {"path": STATICPATH + '/fonts'}),
+    (r'/manifest\.json', Filehandler, {'path': STATICPATH }),
+    (r'/service-workert\.js', Filehandler, {'path': STATICPATH }),
+    (r'/favicon\.ico', Filehandler, {'path': STATICPATH + '/img/icons' }),
```

Comment:
in production, static files will be served from nginx (from dist folder), so these static handlers can be added to tornado only when you have --debug as True (running a local dev server)

Reply:
do you want to put all static files under "/assets" url, like /assets/css/? That way you can use just one line:
```
(r"/assets/(.*)", tornado.web.StaticFileHandler, {"path": STATICPATH}),
```
(except that /favicon.ico has to be at the root, but you can even just ignore it here, since this is for dev-only)